recipes/jupyter_http_over_ws/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "jupyter_http_over_ws" %}
+{% set version = "0.0.8" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b0aa1e7902d3808a698d4853f6dfe12fd02a0d9cb3851db3bf59703106d44a80
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - enum34 ~=1.1  # [py<34]
```

Comment:
You can probably remove this and make this a noarch: python recipe if you don't need Python 2